docs/security.md

```diff
@@ -130,21 +130,16 @@ authentication:
 
 ## Authorization
 
-Trino Gateway supports the following roles in regex string format:
+Trino Gateway supports the following roles in regex string format (for API calls):
 
 - admin : Allows access to the Editor tab, which can be used to configure the
   backends
-
 - user : Allows access to the rest of the website
-
 - api : Allows access to rest apis to configure the backends
 
-Users with attributes next to the role will be giving those privileges the
-users. You can use the preset users defined in the yaml file. 
+Users with attributes matching each role regex will be giving the privileges of that role (admin, user, or api) for Api calls. You can use the preset users defined in the yaml file.
```

Comment:
Please wrap at 80 char columns width so I can review .. but again .. it should be for UI as well